src/SDL2.cs

```diff
@@ -2091,6 +2201,14 @@ SDL_BlendMode blendMode
 			byte b,
 			byte a
 		);
+		
+		public static int SDL_SetRenderDrawColor(
```

Comment:
Oh, I see.  That was added by me as a simple convenience method, because the original API is inconsistent.  (What's the point of declaring a color type and then not using it?)